pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -28,10 +28,10 @@ rec {
   # Policy: use the highest stable version as the default (on our master).
   stable = if stdenv.hostPlatform.system == "x86_64-linux"
     then generic {
-      version = "460.73.01";
-      sha256_64bit = "120ymf59l6nipczszf82lrm2p4ihhqyv2pfwwfg9wy96vqcckc8i";
-      settingsSha256 = "08jh7g34p9yxv5fh1cw0r2pjx65ryiv3w2lk1qg0gxn2r7xypkx0";
-      persistencedSha256 = "040gx4wqp3hxcfb4aba4sl7b01ixr5slhzw0xldwcqlmhpwqphi5";
+      version = "465.27";
```

Comment:
If I looked correctly, 465.27 is from the *New feature branch* and not the *Production branch*. It seems that the policy for this driver has been to stick to the production branch.

Reply:
I didn't properly check it and read it as `stable` on phoronix. Move it into `beta` section.